src/main/obsRecorder.ts

```diff
@@ -4,6 +4,7 @@ import { getAvailableAudioInputDevices, getAvailableAudioOutputDevices } from ".
 import { RecorderOptionsType } from "./recorder";
 import { OurDisplayType } from "./types";
 import { Size } from "electron";
+import { IScene } from "obs-studio-node";
```

Comment:
I've seen this naming a lot but never understood it. What is the I prefix?

Reply:
It's a prefix commonly used for interfaces, mostly to be able to visually distinguish them from classes and types.